app/views/confirmation.njk

```diff
@@ -14,7 +14,7 @@
 
     <p class="govuk-body">We have sent you a confirmation email and your agreement summary.</p>
 
-    <p class="govuk-body">The agreement will be governed by the agreed <a href="/apply/terms">terms and conditions</a></p>
+    <p class="govuk-body">The agreement will be governed by the agreed <a href="{{latestTermsAndConditionsUri}}" target="_blank">terms and conditions</a></p>
```

Comment:
any reason this is now going to a new window?

Reply:
Bit of poetic license 👀  seems weird to me that you would replace a confirmation screen with T and C and then you cannot get back to it. Thought a new tab would be more appropriate

Reply:
removed as per

https://design-system.service.gov.uk/styles/typography/#links:~:text=link%20icon.-,Opening%20links%20in%20a%20new%20tab,-Avoid%20opening%20links